### PR TITLE
Konnect Plus metered billing

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-3353
Changes: 
* Adds new metadata for paid and premium, deleted mentions of Plus in the metadata. 
* Navbar changes - Deletes the old UI instructions for billing
* Redirects
* Added SS of billing dashboard
* Reworked table logic across docs.
* Updates PH tiers and badges and compatibility(Thanks Lena)
image

https://deploy-preview-6099--kongdocs.netlify.app/konnect/compatibility/ 
https://deploy-preview-6099--kongdocs.netlify.app/konnect/account-management/
https://deploy-preview-6099--kongdocs.netlify.app/hub/plugins/license-tiers/#kong-gateway

New table looks like this: 
image

### CODE REVIEW

**Comment by lena-larionova** [comment]
@Guaris the tables for konnect need some work, I think. Having blank spaces is confusing - there should be an X or a checkmark in every cell.

Like Kong Gateway, it's all additive:
* If a plugin is free, it gets a checkmark in every column: Free, Paid, and Premium.
* If a plugin is "paid", it gets an X in the free column and checkmarks in the Paid and Premium columns
* If a plugin is "premium", it gets an X in the free and Paid columns, and a checkmark in the Premium column
* If a plugin isn't available in Konnect at all, all Xs. Otherwise, we will inevitably get people asking us why the plugin is missing from the table.

**Comment by Guaris** [comment]
> * If a plugin is free, it gets a checkmark in every column: Free, Paid, and Premium.

Let's talk about it tomorrow. In the past these were states and that makes sense, but in this case they are associated with a price. If you have free, and you check paid and premium with it then you can't see which tier that plugin belongs to.

**Comment by Guaris** [comment]
@lena-larionova  I updated the logic in the table but it still doesn't feel right. 
image


I think the problem is with the konnect support column. I think red x's across makes sense, but the green check marks is what is throwing me off.

**Review by acgoldsmith — APPROVED** [review]
LGTM and kudos on the tidy, unambiguous solve for which plugins are supported in Konnect!

File: app/konnect/account-management/index.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+{{site.konnect_product_name}} offers [two plans](https://konghq.com/pricing). 
 
-There is no self-serve option for managing Enterprise plans.
-Contact your sales representative to make any changes.
+* {{site.konnect_product_name}} Plus 
+* {{site.konnect_product_name}} Enterprise
+
+{{site.konnect_product_name}} Plus is the simplest way to get started with {{site.konnect_product_name}}, allowing you to only pay for the services you consume. New accounts are automatically given a month of free credits as part of 30-day trial. 
+
+Enterprise is our contract-based option that includes 24x7x365 support and professional services access to help you build and maintain your own custom environment.
```
Comment (acgoldsmith) [inline app/konnect/account-management/index.md:13]:
```suggestion
{{site.konnect_product_name}} Enterprise is our contract-based option that includes 24x7x365 support and professional services access to help you build and maintain your own custom environment. 
```

File: app/konnect/account-management/index.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+## Manage and view plan billing and usage
+
+You can view service, Dev Portal, and API call usage from the [Billing and Usage](https://cloud.konghq.com/settings/billing-settings)
+
+![Billing Page](/assets/images/docs/konnect/billing/billing-and-usage.png)
+
+Here you can review your billable consumption of {{site.konnect_product_name}} and manage billing and payment information.
+
+
+If you want to upgrade to Enterprise, [contact a sales representative](https://konghq.com/contact-sales).
```
Comment (acgoldsmith) [inline app/konnect/account-management/index.md:26]:
```suggestion
If you want to upgrade to {{site.konnect_product_name}} Enterprise, [contact a sales representative](https://konghq.com/contact-sales).
```

File: app/konnect/analytics/index.md [hunk 3]
```diff
@@ -9,11 +9,8 @@ Analytics provides traffic reports to help you track the performance and
 behavior of your APIs and data plane nodes. Use these reports to quickly access key
 statistics, monitor vital signs, and pinpoint anomalies in real time.
 
-Depending on your {{site.konnect_saas}} subscription tier, Analytics retains
-historical data for the following lengths of time:
-* **Free:** 24 hours
-* **Plus:** 3 months
-* **Enterprise:** 1 year
+With **Konnect Plus** Analytics retains historical data for up to 3 months. If you need a longer storage period, [contact a sales representative](https://konghq.com/contact-sales) about upgrading to Enterprise.
```
Comment (acgoldsmith) [inline app/konnect/analytics/index.md:12]:
```suggestion
With **Konnect Plus** Analytics retains historical data for up to 3 months. If you need a longer storage period, [contact a sales representative](https://konghq.com/contact-sales) about upgrading to {{site.konnect_product_name}} Enterprise.
```

**Review by acgoldsmith — COMMENTED** [review]
adding feedback from Ross

**Pull request merged**